storage/blobsasuri.go

```diff
@@ -44,7 +44,7 @@ func (b *Blob) GetSASURIWithSignedIPAndProtocol(expiry time.Time, permissions st
 		signedResource = "b"
 	}
 
-	protocols := "https,http"
+	protocols := ""
```

Comment:
Why we don't need "http" anymore?

Reply:
The protocols are set to `https,http` by default, so setting them explicitly is not really needed.
On the other hand, this is also useful to generate exactly the same SAS as the Storage Explorer does.